examples/voip/Server.cpp

```diff
@@ -85,7 +85,7 @@ class NetworkAudioStream : public sf::SoundStream
         // (don't forget that we run in two separate threads)
         {
             sf::Lock lock(m_mutex);
-            m_tempBuffer.assign(m_samples.begin() + static_cast<std::vector<sf::Int64>::difference_type>(m_offset), m_samples.end());
+            m_tempBuffer.assign(m_samples.begin() + static_cast<std::vector<sf::Int16>::difference_type>(m_offset), m_samples.end());
```

Comment:
```suggestion
            m_tempBuffer.assign(m_samples.begin() + static_cast<decltype(m_samples)::difference_type>(m_offset), m_samples.end());
```

Maybe this expresses the intent better?

Reply:
You are right. However that is C++11 and I target 2.6.x branch which uses C++03.

Reply:
Back in SFML 3 land I would like to eventually find more of these places where the names of types are being repeated and replace them with `decltype`.

Reply:
Except this is targeting 2.6.x thus no decltype for you 😉